src/nepiada/env/nepiada.py

```diff
@@ -94,6 +95,16 @@ def render(self):
             # Temporary to print grid for debug purposes until we have a better way to render.
             self.world.grid.render_grid()
             self.world.graph.render_graph(comm=False)
+
+            # This is a debugging code
+            # TODO: Remove later
```

Comment:
I have kept this here, until I get a unit test up

Reply:
Maybe just keep it commented out for the version we merge

Reply:
Done!